src/Components/Main/Form/Form.js

```diff
@@ -0,0 +1,17 @@
+import TextArea from "./TextArea";
+import DateComponent from "../DateComponent";
+import "./Form.css";
+
+export default function Form() {
+  return (
+    <form className="form">
+      <h2 className="h2">
+        NEW ENTRY - TODAY,
+        <DateComponent />
+      </h2>
+      <TextArea name="motto" labelText="Motto" />
+      <TextArea name="notes" labelText="Notes" />
```

Comment:
labelText kenne ich gar nicht. Muss ich mir mal anschauen, aber ich hätte es in einer extra Zeile mit <label htmlFor="...">Motto<label> oder so geschrieben

Reply:
Ach ich sehe gerade, dass du es in der TextArea.js so gemacht hast 👍